middlewares/validators/user.js

```diff
@@ -320,7 +320,24 @@ const validateGenerateUsernameQuery = async (req, res, next) => {
     res.boom.badRequest("Invalid Query Parameters Passed");
   }
 };
-
+const migrationsValidator = async (req, res, next) => {
+  const { action, skip, limit } = req.query;
+  const schema = joi
+    .object()
+    .strict()
+    .keys({
+      skip: joi.number(),
+      action: joi.string().valid("adds-github-id").required(),
+      limit: joi.number().min(1).max(500).required(),
+    });
```

Comment:
Can you please follow conventions mentioned in our docs: https://github.com/Real-Dev-Squad/docs/tree/main/docs/conventions/api

Reply:
done